src/re/main.c

```diff
@@ -1166,6 +1182,11 @@ main(int argc, char *argv[])
 		return 0;
 	}
 
+	if (generate_bounds > 0) {
+		int r = fsm_generate_matches(fsm, generate_bounds, fsm_generate_cb_printf, NULL);
+		return r;
```

Comment:
why the variable?

Reply:
Oh, because I was initially freeing the `fsm` before exiting, then realized that the `atexit` handler was already doing it. I'll remove the unnecessary variable.

Reply:
Removed in 3ae4c9be.

Reply:
aha! thanks